test/Parse/interpolation_tuple_errors.swift

```diff
@@ -0,0 +1,23 @@
+// RUN: not %target-swift-frontend -typecheck -verify %s
+
+let str = "a"
+
+// Test with too many interpolated values.
+func papaBear() {
+  _ = "\(str, str)" // expected-error{{interpolations should be a single expression}} expected-note {{insert parentheses to form a tuple}} {{9-9=(}} {{19-19=)}}
+}
+
+// Test with too few interpolated values.
+func mamaBear() {
+  _ = "\()" // expected-error{{expected expression in list of expressions}}
```

Comment:
Any way that we can also make this error less cryptic as part of this change? It’s essentially nonsense right now, at least to me.

Reply:
It was surprisingly nontrivial, but I've just pushed changes to do it. The new version emits the same "interpolations should be a single expression" error message for empty interpolations, but provides a different fix-it telling the user to delete the empty interpolation.